src/events/crawler/scrapers/USA/VA/index.js

```diff
@@ -126,25 +125,25 @@ const scraper = {
     'Rockbridge County',
     'Rockingham County',
     'Russell County',
-    'Salem city',
+    'Salem County',
     'Scott County',
     'Shenandoah County',
     'Smyth County',
     'Southampton County',
     'Spotsylvania County',
     'Stafford County',
-    'Staunton city',
-    'Suffolk city',
+    'Staunton County',
+    'Suffolk County',
     'Surry County',
     'Sussex County',
     'Tazewell County',
-    'Virginia Beach city',
+    'Virginia Beach County',
     'Warren County',
     'Washington County',
-    'Waynesboro city',
```

Comment:
Do we need to canonicalize these names in the `coronavirus-data-sources` too? For example, Waynseboro appears as "city" in the geojson but "County" in the population csv.

```
$ grep Waynesboro */*
geojson/usa-counties.json:        "name": "Waynesboro County, VA"
population/population-usa-counties.csv:"Waynesboro city, VA",22628
```

Reply:
Yes, already put in a PR for that.